### PR TITLE
Document Giphy integration

### PR DESCRIPTION
**Pull request by panoschal**
Associated PR: https://github.com/matrix-org/matrix-react-sdk/pull/5814

Signed-off-by: Panagiotis Chalimourdas <panagiotis4531@gmail.com>

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Comment by SimonBrandner** [comment]
@panoschal, I think the merge broke your PR :P 
Screenshot_20210509_204042

**Comment by tsukasagenesis** [comment]
If someone can look into it

**Review by t3chguy — CHANGES_REQUESTED** [review]
Needs DCO sign-off (see https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.rst#sign-off) and the tests passing

**Comment by Palid** [comment]
Is this still relevant? @panoschal
New sign-off link: https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.md#sign-off

**Comment by SimonBrandner** [comment]
> Is this still relevant? @panoschal

Definitely, the react-sdk PR is waiting for design/product review though, so this is quite stuck... Perhaps someone could give them a nudge

File: docs/labs.md [hunk 1]
```diff
@@ -130,6 +130,12 @@ encrypted on the server, as described in [MSC2697](https://github.com/matrix-org
 Enables UI for turning on "do not disturb" mode for the current device. When DND mode is engaged, popups
 and notification noises are suppressed. Not perfect, but can help reduce noise.
 
+## Giphy Integration (`feature_giphy_integration`)
+
+Giphy API integration, that allows the users to send GIFs by searching with keywords, and selecting the one they like. It displays a new button in the composer next to the stickers and emojis. The GIFs are sent as files/attachments in the conversation.
+
+Associated PR: https://github.com/matrix-org/matrix-react-sdk/pull/5814
```
Comment (SimonBrandner) [inline docs/labs.md:137]: I don't think we usually link to a specific PR. You might do some follow up PRs and it doesn't really make sense to link them all here. What we usually do though is link the PR in this PR's description

**Review by SimonBrandner — CHANGES_REQUESTED** [review]

**Comment by SimonBrandner** [comment]
Your CI is failing because it using _your_ develop branches from the other layers. You'll have to merge and push on these too

**Review by SimonBrandner — APPROVED** [review]
LGTM - thanks!

**Comment by panoschal** [comment]
Great! I signed in the PR description.

**Comment by SimonBrandner** [comment]
Marking as blocked since the react-sdk PR hasn't been merged yet

**Comment by t3chguy** [comment]
The associated PR this was for got closed

**Pull request closed without merging**